t/lib/server.lua

```diff
@@ -112,10 +112,6 @@ function _M.mock_skywalking_v2_instance_register()
     ngx.say(json_encode(ret))
 end
 
-function _M.mock_skywalking_v2_instance_heartbeat()
-    ngx.say('skywalking heartbeat ok')
-end
-
```

Comment:
these codes should not remove.

Reply:
Need to revert these codes, then the test case will run successfully.

```
Test Summary Report
-------------------
t/plugin/skywalking.t                (Wstat: 256 Tests: 30 Failed: 1)
  Failed test:  10
  Non-zero exit status: 1
Files=120, Tests=4005, 1041 wallclock secs ( 1.02 usr  0.16 sys + 152.60 cusr 29.88 csys = 183.66 CPU)
Result: FAIL
##[error]Process completed with exit code 1.
```

Reply:
reverted in latest commit.